example-deployment/README.md

```diff
@@ -246,6 +246,60 @@ deploy it
 .bin/minikube kubectl -- apply -f k8s/dashboard.yaml
 ```
 
+## [MinIO](https://github.com/minio/minio)
+MinIO is a High-Performance Object Storage released under Apache License v2.0. MinIO has several uses but in our case, we will use MinIO to store logs.
+
+``` bash
+# connected to internet
+docker pull minio/minio:RELEASE.2021-03-10T05-11-33Z
+
+# connected to airgap network
+docker save minio/minio:RELEASE.2021-03-10T05-11-33Z | bash -c 'eval $(.bin/minikube docker-env) && docker load'
+
+.bin/minikube kubectl -- apply -f k8s/minio.yaml
+```
```

Comment:
The docker instructions are not normally needed, when you deploy to k8s, it should auto fetch the images.

For airgapped deployment, I think it's better to direction people to the airgapped section instead of assuming (and providing) only the airgapped instructions.

Reply:
Yeah, you're right, I'll do that!. Thanks!